src/styles/s-forms.css

```diff
@@ -115,7 +115,7 @@ input:disabled {
 }
 
 .accordion > .card {
-  overflow: visible;
+  overflow: visible !important;
```

Comment:
IMHO, it is not good practice to use !important , because it is then harder to reuse CSS style ... , but I am amateur here:). What do u think ? Is it the only way to solve it ?

Reply:
Yes you are right, !important is bad practice as it overrides styles. I thought it is the only way to override Bootstrap default style, but actually style import order matters: https://stackoverflow.com/questions/20721248/how-can-i-override-bootstrap-css-styles

I made new commit :)